src/views/LimitOrders/hooks/useGelatoLimitOrdersHistory.ts

```diff
@@ -36,29 +55,30 @@ const useOpenOrders = (turnOn: boolean): Order[] => {
     startFetch ? ['gelato', 'openOrders'] : null,
     async () => {
       try {
-        const orders = await gelatoLimitOrders.getOpenOrders(account.toLowerCase(), false)
+        const orders = await gelatoLimitOrders.getOrders(account.toLowerCase(), false)
```

Comment:
Any reason to use `getOrders` instead of `getOpenOrders` here?

Reply:
This is for to detect open order is executed, otherwise it will stay on open orders on the same sevice if user doesnt change the tab to order history. But i have a optimised solution which we dont have to fetch all orders, i will apply soon